server/api.js

```diff
@@ -284,8 +290,8 @@ router.post("/tender", async (req, res) => {
 		await client.query("BEGIN");
 
 		const insertTenderQuery = `
-		INSERT INTO tender (title, announcement_date, deadline, description, closing_date, status)
-		VALUES ($1, $2, $3, $4, $5, $6) RETURNING id
+		INSERT INTO tender (title, announcement_date, deadline, description, closing_date, status,  buyer_id, creation_date, last_update, no_of_bids_received)
+		VALUES ($1, $2, $3, $4, $5, $6, $7, $8, CURRENT_DATE, now(), $9) RETURNING id
```

Comment:
Why do you use `CURRENT_DATE` for one date and `now()` for the other one?

Reply:
Because for the last_update I wanted both date and time while for creation date only the date thats why I used the current_date

Reply:
I hadn't noticed they are two different types. We should probably have used the same type for both, but it's too late for that. You can ignore this comment.